e2e/json_util_test/test_load_from_url/data.json

```diff
@@ -0,0 +1,9 @@
+{
+    "name": "Michael",
+    "age": 41,
+    "children": [
+        "Selina",
+        "Rana",
+        "Selma"
+    ]
+}
```

Comment:
Can this be deleted now that you have it on MG download page

Reply:
Yes, you're right, I'll delete it